bin/pycbc_inspiral

```diff
@@ -25,6 +25,7 @@ from pycbc.types import TimeSeries, FrequencySeries, zeros, float32, complex64
 import pycbc.fft.fftw, pycbc.version
 import pycbc.opt
 import pycbc.weave
+import pycbc.injcutter
```

Comment:
How about?

--filter-rejector-chirp-time-threshold

and then I name the module FilterRejector and object names to filterrejector. There has to be some balance between length of these names and readability and I think this is an okay compromise. If it's in the injection module as Alex suggests then:

from injection import FilterRejector

should be clear?